Tests/Tests.Shared/ConfigurationTests.cs

```diff
@@ -66,7 +83,7 @@ public void PathIsCanonicalised()
             // Assert
             Assert.That(Path.IsPathRooted(config.DatabasePath));
             Assert.That(config.DatabasePath, Does.EndWith(Path.Combine("Documents", "fred.realm")));
-            Assert.IsFalse(config.DatabasePath.Contains(".."));  // our use of relative up and down again was normalised out
+            Assert.That(config.DatabasePath.Contains(".."), Is.False);  // our use of relative up and down again was normalised out
```

Comment:
Could probably be `Assert.That(config.DatabasePath, Does.Not.Contain(".."))`. I think.